mmdet/models/anchor_heads/atss_head.py

```diff
@@ -67,8 +71,12 @@ def __init__(self,
         octave_scales = np.array(
             [2**(i / scales_per_octave) for i in range(scales_per_octave)])
         anchor_scales = octave_scales * octave_base_scale
```

Comment:
`octave_base_scale` and `scales_per_octave` can also be arguments of anchor generators